racetime/templates/racetime/race/aside.html

```diff
@@ -24,6 +24,12 @@
             Reconnecting to chat…
         </span>
     </div>
+    <div class="scrollwarning" title="Click here to scroll to the bottom" onwheel="UpdateScroll()">
+        <span>
+            <i class="material-icons">history</i>
+            Chat Paused Due to Scroll
```

Comment:
Use sentence case - "Chat paused due to scroll"

Reply:
I was using the same casing twitch was in that image. Cool with swapping it though.